# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/fake/minisip/gtk-include-path-fix.diff
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ././minisip/gui/gtkgui/eggtrayicon.cxx.orig	2005-02-22 13:18:58.417257256 +0100
+++ ././minisip/gui/gtkgui/eggtrayicon.cxx	2005-02-22 13:19:15.680632824 +0100
@@ -20,7 +20,7 @@
 
 #ifndef WIN32
 #include <string.h>
-#include <gtk-2.0/gdk/gdkx.h>
+#include <gdk/gdkx.h>
 //#include<gtkmm.h>
 #include "eggtrayicon.h"
 
--- ././minisip/gui/gtkgui/eggtrayicon.h.orig	2005-02-22 13:19:06.194075000 +0100
+++ ././minisip/gui/gtkgui/eggtrayicon.h	2005-02-22 13:19:29.178580824 +0100
@@ -22,8 +22,8 @@
 #define __EGG_TRAY_ICON_H__
 
 #ifndef WIN32
-#include <gtk-2.0/gtk/gtkplug.h>
-#include <gtk-2.0/gdk/gdkx.h>
+#include <gtk/gtkplug.h>
+#include <gdk/gdkx.h>
 
 G_BEGIN_DECLS
 
